src/main/java/com/google/sps/data/SessionContext.java

```diff
@@ -56,4 +73,21 @@ public String getLoggedInUserId() {
   public boolean isUserLoggedIn() {
     return userService.isUserLoggedIn();
   }
+
+  public boolean isUserExistingInDatastore() throws Exception {
```

Comment:
- Does this method throw an exception?
- Could you rename this to userProfileExists()?

Reply:
No it does not.